addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -871,7 +877,6 @@ var ImageWidget = FileWidget.extend({
     _updateAddUrlUi: function (emptyValue, isURL, isImage) {
         this._super.apply(this, arguments);
         const warning = isURL && !isImage;
-        this.$urlWarning.toggleClass('d-none', !warning);
```

Comment:
I don't understand this change either.

Reply:
I was thinking why to show the warning sign when i have purposely deleted all the url. i believe there is no need of warning or something like that we can simply change btn-primary to btn-secondry
exactly like we are freshly entering the url.
@dmo-odoo what is your view on it?